communication/ftp/send/send-file-using-ftp.yml

```diff
@@ -0,0 +1,28 @@
+rule:
+  meta:
+    name: send file using FTP
+    namespace: communication/ftp/send
+    authors:
+      - michael.hunhof@mandiant.com
+      - anushka.virgaonkar@mandiant.com
+    scope: function
+    mbc:
+      - Communication::FTP Communication::Send File [C0004.001]
+      - Communication::FTP Communication::WinINet [C0004.002]
+    examples:
+      - Practical Malware Analysis Lab 20-02.exe_:0x401380
+  features:
+    - or:
+      - and:
+        - api: wininet.FtpPutFile
+        - optional:
+          - or:
+            - api: wininet.FtpSetCurrentDirectory
+            - and:
+              - api: wininet.InternetConnect
+              - number: 0x15 = IPPORT_FTP
+      - and:
+        - api: System.Net.WebRequest::Create
+        - class: System.Net.FtpWebRequest
+        - string: "STOR"
+        - property: System.Net.WebRequest::Method
```

Comment:
Are the `api`, `class`, and `property` expected to be extracted from the same instruction?
Is there duplication in this case?

Reply:
No, they are extracted from different instructions.

Reply:
Ok, I'm having trouble following this. Can you include a comment of an example code snippet?

Reply:
![image](https://user-images.githubusercontent.com/41755334/184044322-517d2e15-a4ad-4ca8-9fc2-a9c16952002c.png)

Reply:
Thanks, so we could remove `class: System.Net.FtpWebRequest` and still be good?

Reply:
Unrelated, can we do this?
```yaml
instruction:
  - property: System.Net.WebRequest::Method
  - string: STOR
```

Reply:
Yes, we can remove `class: System.Net.FtpWebRequest`

![image](https://user-images.githubusercontent.com/41755334/184260491-7006c357-042d-44f0-a177-b66851885553.png)
Since the features are extracted from different instructions, we won't be getting a rule match.

Reply:
Thanks. And yes, never mind my `instruction` idea :)